DataWorkerService.Tests/AutomationChecks/Matches/MatchGameCountCheckTests.cs

```diff
@@ -93,8 +93,11 @@ public void Check_GivenVerifiedGamesCount_IsEven_FailsWith_UnexpectedGameCount()
     [Theory]
     [InlineData(0, 0, false, MatchRejectionReason.NoGames)]
     [InlineData(0, 2, false, MatchRejectionReason.NoValidGames)]
-    [InlineData(4, 3, false, MatchRejectionReason.UnexpectedGameCount)]
-    [InlineData(10, 0, false, MatchRejectionReason.UnexpectedGameCount)]
+    [InlineData(1, 1, false, MatchRejectionReason.UnexpectedGameCount)]
+    [InlineData(2, 2, false, MatchRejectionReason.UnexpectedGameCount)]
+    [InlineData(3, 3, true, MatchRejectionReason.None)]
+    [InlineData(4, 3, true, MatchRejectionReason.None)]
+    [InlineData(10, 0, true, MatchRejectionReason.None)]
```

Comment:
Would it make sense to have another test for more rejected games than verified ones?

`[InlineData(3, 7, true, MatchRejectionReason.None)]`